tests/test_utils.py

```diff
@@ -58,3 +60,46 @@ def test_upload_dir_to_s3(tmp_path):
                 call(tmp_path / 'subdir1/subdir3/bar.txt', 'myBucket', 'myPrefix/subdir1/subdir3/bar.txt'),
             ]
         )
+
+
+@mock_aws
+def test_list_files_in_s3(s3_bucket):
+    prefix = 'geotiffs'
+    geotiffs = [
+        f'{prefix}/my-file1.tif',
+        f'{prefix}/my-file2.tif',
+        f'{prefix}/my-file3.tif',
+    ]
+
+    for tif in geotiffs:
+        ut.S3_CLIENT.put_object(Bucket=s3_bucket, Key=tif)
+
+    files = ut.list_files_in_s3(s3_bucket, prefix)
+
+    assert len(files) == len(geotiffs)
+    assert files[0]['Key'].endswith('.tif')
```

Comment:
```suggestion
    assert len(files) == 3
   assert files[0]['Key'] == my-file1.tif
   assert files[1]['Key'] == my-file2.tif
   assert files[2]['Key'] == my-file3.tif
```

Reply:
Added this, needed to add prefix to filenames